cmd/query/main_test.go

```diff
@@ -37,6 +40,9 @@ func TestSliceHappyHTTPResponse(t *testing.T) {
 				Direction: "crossline",
 				Lineno:    10,
 				Sas:       "n/a",
+				Bounds: []testBound{
+					{ Direction: "inline", Lower: 1, Upper: 3 },
+				},
 			},
 		},
```

Comment:
Wondering if e2e tests should be always updated with optional parameters or not. 
Should keep them and their json spelling locked.